framework/components/ecl-footers/_footers.scss

```diff
@@ -1,142 +1,122 @@
-/*
-Footer
-
-Site-wide footer component
-
-Markup: html/footer.html
-
-Style guide footer
-*/
-
-.footer {
-  background-color: $blue-medium;
+/**
+ * Footers component
+ * @define footers ; weak
```

Comment:
FYI, I had to make it "weak" because of `.ecl-*** a` selectors. Read more: https://github.com/postcss/postcss-bem-linter#conformance-tests